src/gg.regression.unity.bots/Runtime/Scripts/IRGBot.cs

```diff
@@ -0,0 +1,5 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public interface IRGBot { }
```

Comment:
I wonder if we should make this an abstract class... maybe we can always change it later, but I imagine there may be some built-in, implemented functions we may want to add at some point.

Reply:
I would vote against that.  You will already be inheriting from MonoBehaviour, keeping this as an interface would seem more appropriate

Reply:
@vontell I could see either way, I don't think it will really affect the dev experience so more of an architecture question

`public class MyTestBot: MonoBehavior, IRGBot`
`public class MyTestBot: MonoBehavior, RGBot`